Fix Next.js Deprecation Warnings in Image Component and Config

midhunann opened the pull request:
When the project is run, there are two deprecation warnings which are not alligned with Next.js best practice.
1. Deprecation Warning in Image Component
2. Deprecation Warning for images.domains Configuration

Fixed both of them
before/
![Screenshot 2025-01-11 143711](https://github.com/user-attachments/assets/bd5c4d85-938e-4066-9c4e-d4a781dbc233)

after/
![Screenshot 2025-01-11 145815](https://github.com/user-attachments/assets/ff4b00ab-cafa-4975-a17c-101e49dc572c)

Please do review this PR and if there has to be any changes do let me know.
Thank You!

Ashrockzzz2003 reviewed (commented):
@midhunann But then, using `next/legacy/image` means, we're using a deprecated package. Instead can you make work with `next/image`.

midhunann commented:
### **Changes Made**

1. Removed the deprecated package to a newer one.

before/
```tsx
import Image from "next/legacy/image";
```
after/
```tsx
import Image from "next/image";
```
2. Changed the format of Image Properties

before/
```tsx
<Image src="/example .png" layout="fill" objectFit="cover" />
```

after/
```tsx
<Image src="/example.png" fill style={{ objectFit: 'cover' }} />
```

Ashrockzzz2003 reviewed (approved):
LGTM

Ashrockzzz2003 commented:
/bounty 35 @midhunann

midhunann commented:
Thank You!! @Ashrockzzz2003
